style.css

```diff
@@ -0,0 +1,61 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.first-page {
+  background-image: url(./images/background.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  width: 340px;
```

Comment:
### OPTIONAL 
- [ ] Kindly remove or adjust the width, you can  use margin  to get the appropriate size as seen in the[ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) 
- [ ] kindly add the background image to the appropriate section, check this [article](https://www.w3schools.com/cssref/css3_pr_background-size.asp) on how to style background image with css